### DISCUSSION TITLE
7.5.0 release

### DISCUSSION THREAD

**Discussion by ricardofontanelli** [lcobucci/di-builder#543] [General]
Hi there, do you have any plans to release the 7.5.0 tag? I was looking to an easy way to upgrade some apps to PHP 8.3.

**Comment by lcobucci** [comment]
Is there anything preventing you from going to v8.0 instead?

There BC-breaks but they should be really contained.

**Reply by ricardofontanelli** [reply]
In my specific case, the BC (not sure if it would impact anything we already have) + dependabot would help me to open all PRs since it's not a major version update 😅. I'm assuming 7.4 to 7.5 would be basically support to latest PHP version.

**Reply by lcobucci** [reply]
At the moment there is no plan to release v7.5, as 7.x feature support ends with the 8.x release.

I understand that having a minor release can facilitate the semantics of dependency bump automation, but that's usually related to organisational constraints than technical feasibility... 

I'm a bit conflicted with allocating my limited oss time to address this.
